cluster/examples/kubernetes/ceph/cluster.yaml

```diff
@@ -143,3 +143,15 @@ spec:
 #        storeType: filestore
 #    - name: "172.17.4.301"
 #      deviceFilter: "^sd."
+  # Override Ceph configs. These can be overridden by the user at runtime using the Ceph CLI and
+  # and will be reset by Rook when the operator restarts.
```

Comment:
What does "reset" mean here? That those options are removed from the CRD when the operator restarts?